client/keys/show.go

```diff
@@ -35,9 +35,9 @@ const (
 // ShowKeysCmd shows key information for a given key name.
 func ShowKeysCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "show [name [name...]]",
-		Short: "Show key info for the given name",
-		Long: `Return public details of a single local key. If multiple names are
+		Use:   "show [name_or_address [name_or_address...]]",
+		Short: "Retrieve key information for the given by name or address",
+		Long: `Return public details of a single local key. If multiple names or addresses are
```

Comment:
public details?

Reply:
Well, don't know much. It was there already :-/
But I can amend it for sure!